### PR TITLE
Create xyz.txt

### PR DESCRIPTION
**Pull request by vancuongngo**

### CODE REVIEW

File: xyz.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+xyz
```
Comment (cuongngo-kry) [inline xyz.txt:1]: one

Comment (cuongngo-kry) [reply xyz.txt:1]: two

**Review by cuongngo-kry — COMMENTED** [review]
hello

File: xyz.txt (hunk 1 above)
Comment (vancuongngo) [reply xyz.txt:1]: ok

**Review by cuongngo-kry — CHANGES_REQUESTED** [review]
you need to change

**Comment by vancuongngo** [comment]
ok, I will change

**Comment by vancuongngo** [comment]
ok ok

**Comment by vancuongngo** [comment]
ok ok ok

**Review by cuongngo-kry — APPROVED** [review]
good

**Review by cuongngo-kry — CHANGES_REQUESTED** [review]
nto good

**Review by cuongngo-kry — APPROVED** [review]
good

**Review by cuongngo-kry — APPROVED** [review]
ok

**Review by cuongngo-kry — COMMENTED** [review]
aha

**Pull request merged**